vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -85,9 +85,11 @@ if isMarinerOrAzureLinux "$OS"; then
     installFIPS
   fi
 else
-  # Enable ESM for 18.04 and FIPS only
+  # Enable ESM only for 18.04, 20.04, and FIPS
   if [[ "${UBUNTU_RELEASE}" == "18.04" ]] || [[ "${UBUNTU_RELEASE}" == "20.04" ]] || [[ "${ENABLE_FIPS,,}" == "true" ]]; then
+    set +x
     autoAttachUA
```

Comment:
we should renamed, since auto-attach if the name of the feature with the subID in an allow-list,  auto will automatically fetch the Token based on the SubID Lookup.

Reply:
fixed